packages/frontend/editor-ui/src/components/SourceControlPullModal.ee.test.ts

```diff
@@ -198,10 +199,13 @@ describe('SourceControlPushModal', () => {
 			},
 		});
 
-		// For credential files, there should be no additional buttons in the item actions
-		const itemActions = container.querySelector('[class*="itemActions"]');
-		const buttons = itemActions?.querySelectorAll('button');
-		expect(buttons).toHaveLength(0);
+		// Click on credentials tab to show credential items
+		await userEvent.click(getByText('Credentials'));
+
+		// For credential files, there should be no diff buttons (only badges in the badges container)
+		const badges = container.querySelector('[class*="badges"]');
+		const buttons = badges?.querySelectorAll('button');
+		expect(buttons?.length || 0).toBe(0);
```

Comment:
Is there a case where we don't get the buttons? The expectation seems strange to just check that 0 is 0 if it fails.

Reply:
Yes, exactly